Topics/Development_Process/Conda/Managing_environments.md

```diff
@@ -0,0 +1,113 @@
+## Managing environments
```

Comment:
also only one '#' here, same thing for the rest of the document

Reply:
Edited.